examples/pytorch/nlp/huggingface_models/question-answering/quantization/ptq_static/ipex/README.md

```diff
@@ -32,6 +34,79 @@ python run_qa.py
 >
 > /path/to/checkpoint/dir is the path to finetune output_dir
 
+## 2. Quantization with XPU
+Please build an IPEX docker container with following steps. Please also refer to the [official guide](https://github.com/intel/intel-extension-for-pytorch/tree/xpu-master/docker).
+#### 2.1 Build Container and Environment Variables
+```bash
+wget https://raw.githubusercontent.com/intel/intel-extension-for-pytorch/xpu-master/docker/Dockerfile.xpu
+wget https://raw.githubusercontent.com/intel/intel-extension-for-pytorch/xpu-master/docker/build.sh
+./build.sh xpu-flex
+
+export IMAGE_NAME=intel/intel-extension-for-pytorch:xpu-flex
+export VIDEO=$(getent group video | sed -E 's,^video:[^:]*:([^:]*):.*$,\1,')
+export RENDER=$(getent group render | sed -E 's,^render:[^:]*:([^:]*):.*$,\1,')
+test -z "$RENDER" || RENDER_GROUP="--group-add ${RENDER}"
+```
+
+#### 2.2 Run Container
+```bash
+docker run --rm \
+    -v .:/workspace \
+    --group-add ${VIDEO} \
+    ${RENDER_GROUP} \
+    --device=/dev/dri \
+    --ipc=host \
+    -e http_proxy=$http_proxy \
+    -e https_proxy=$https_proxy \
+    -e no_proxy=$no_proxy \
+    -it $IMAGE_NAME bash
+```
+
+#### 2.3 Environment Settings
+Source installed torch is required:
+```bash
+git clone https://github.com/intel-innersource/frameworks.ai.pytorch.private-gpu.git --depth=1
```

Comment:
This is internal link, is there an available external link?

Reply:
switch to public repo w/ tutorial link